specification/columns/pricingsubcategory.md

```diff
@@ -0,0 +1,70 @@
+# Pricing Subcategory
+
+Pricing Subcategory describes the pricing model used for a charge within a specific [Pricing Category](#pricingcategory). It can be useful for distinguishing between charges that share the same Pricing Category, but have different pricing rules that may result in a lower prices, like flat-rate compared to volume-based discounts.
```

Comment:
Not entirely sure if the words "pricing model" are self-explanatory. Is there possibly another way to say that might be less 'left to interpretation' ?

Reply:
Hmm... 🤔 Will put some thought into it. I used the same thing we have for PricingCategory, but we can update both places. Anyone else have thoughts or ideas?

Reply:
I find that pricing model is a common concept, but to me the issue is PricingCategory and PricingSubcategory do not actually describe the pricing model but refer to categories and sub categories of used pricing models.

So maybe simply:
> Pricing Subcategory describes the sub category in the pricing model used for a charge within a specific Pricing Category

If the term pricing model is too abstract, maybe the term pricing type could be used instead.